packages/extension-ui/src/Popup/ImportSeed/SeedAndPath.tsx

```diff
@@ -1,6 +1,8 @@
 // Copyright 2019-2023 @polkadot/extension-ui authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+/* eslint-disable react/jsx-newline */
```

Comment:
Why's that?

Reply:
![image](https://user-images.githubusercontent.com/50427204/219307454-36788d6c-703c-48cc-9f4b-6ab392412169.png)

Reply:
Maybe you can try to configure it with config options to allow this type of behaviour? 
**https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-newline.md**